packages/api-client/src/libs/create-client.ts

```diff
@@ -13,6 +13,11 @@ import type { LiteralUnion } from 'type-fest'
 import { type Component, createApp, watch } from 'vue'
 import type { Router } from 'vue-router'
 
+/**
+ * The layout of the client
+ */
+export type ClientLayout = 'modal' | 'web' | 'desktop'
```

Comment:
Not sure if the type should live here or with the hook but it's typed now!

Reply:
@amritk @hanspagel

Reply:
Love the hook but should we change desktop? Its not really clear as web can be desktop too. How about app or even electron? Type should live with the hook imo, always keep it close to the meat

Reply:
In my mind desktop !== web whereas app can be a web, desktop or mobile app. We use toDesktop to which seems to call electron apps desktop apps right? 

![Google Chrome-2024-11-08-18-40-22@2x](https://github.com/user-attachments/assets/a077dd6c-4eca-4373-8aa9-477e633fe62f)

Reply:
I can move the types to the hook though 🫡